charts/pulsar-resources-operator/Chart.yaml

```diff
@@ -30,13 +30,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: v0.4.3
+version: v0.4.4
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "v0.4.3"
+appVersion: "v0.4.4"
```

Comment:
@mingmcb `appVersion` means the operator image version here. 
So for your case,  should not change the `appVersion` here cause your change is only about the chart template and won't need a new operator image.